MAUI/Scheduler/drag-and-drop-appointments.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: Appointment drag and drop in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about Appointment drag and drop support in Syncfusion .NET MAUI Scheduler(SfScheduler) control.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Appointment Drag and Drop in .NET MAUI Scheduler
+Appointments can be rescheduled using the drag and drop operation. To perform drag-and-drop operations within the schedule, enable the `AllowAppointmentDrag` property of SfScheduler.
+
+{% tabs %}
+{% highlight xaml %}
+<scheduler:SfScheduler x:Name="scheduler" View="Week" AllowAppointmentDrag="True">        
```

Comment:
Mention by default true, you can disable by chnging false
AllowAppointmentDrag - change example code to false

Reply:
changed content